docs/play_kube_support.md

```diff
@@ -0,0 +1,152 @@
+# Podman Play Kube Support
+
+This document outlines the kube yaml fields that we currently support with the **podman play kube** command.
```

Comment:
```suggestion
This document outlines the kube yaml fields that are currently support by the **podman play kube** command.
```

Reply:
Nice eye Valentin, no pronouns in man pages please!